modules/external_commands/src/commands.cc

```diff
@@ -570,12 +570,13 @@ int process_passive_service_check(
     real_host_name = host_name;
   else {
     for (host_map::iterator
-           it(host::hosts.begin()),
+           itt(host::hosts.begin()),
            end(host::hosts.end());
-         it != end;
-         ++it) {
-      if (it->second->get_address() == host_name) {
-        real_host_name = it->first.c_str();
+         itt != end;
+         ++itt) {
+      if (itt->second->get_address() == host_name) {
+        real_host_name = itt->first.c_str();
+        it = itt;
```

Comment:
it = itt looks not necessary, it looks to be unused after.

Reply:
I'm agree but it is just to keep an iterator that has a sense